README.md

```diff
@@ -1,12 +1,36 @@
-Repository containing all of BetterPvP's plugins for 1.19.
+Repository containing all of BetterPvP's plugins for 1.20.1.
 
 ## Dependencies:
 - Java 17
 - PaperSpigot
 - A MySQL server
+- [MythicMobs](https://www.spigotmc.org/resources/mythicmobs.5702)
 - [ProtocolLib](https://www.spigotmc.org/resources/protocollib.1997)
 - [LibsDisguises](https://www.spigotmc.org/resources/libs-disguises-free.81)
 
 
-## Setup
-Instructions will become available later
\ No newline at end of file
+## Development Setup
+1. Clone this repository and open it with your IDE of choice:
+    ```bash
+    git clone https://github.com/Mykindos/BetterPvP.git
+    ```
+2. Compile the project using the Gradle task `reobfJar` to properly shade all project dependencies:
+   **NOTE: You must run `reobfJar` to properly obfuscate the Mojang mappings.**
+    ```bash
+    ./gradlew reobfJar
+    ```
+   
+3. Copy the generated jar files from all modules (`./build/libs/`) into your server's `plugins` folder.
+4. In your plugins folder, make sure to also include ProtocolLib, LibsDisguises, and MythicMobs as they are required dependencies.
+5. Create a MySQL database and import all `.sql` files located in the resource folders of each module.
```

Comment:
You only need to create the database, the rest is handled by flyway

database.getConnection().runDatabaseMigrations(getClass().getClassLoader(), "classpath:champions-migrations", databasePrefix);

^ above code is in onEnable for each plugin

This is also where the ${tablePrefix} gets populated when executing the SQL files, so your changes to the sql files will break this automatic feature